core/src/test/java/ai/timefold/solver/core/testconstraint/TestConstraintFactory.java

```diff
@@ -33,6 +34,11 @@ public SolutionDescriptor<Solution_> getSolutionDescriptor() {
         throw new UnsupportedOperationException();
     }
 
+    @Override
+    public <A> @NonNull UniConstraintStream<A> forEachIncluding(@NonNull Class<A> sourceClass, ForEachInclude... includes) {
+        throw new UnsupportedOperationException();
+    }
+
     @Override
     public <A> @NonNull UniConstraintStream<A> forEachIncludingUnassigned(@NonNull Class<A> sourceClass) {
         throw new UnsupportedOperationException();
```

Comment:
Arguably the override is unnecessary, if we give the original method a `default` implementation. IMO it's cleaner, because it doesn't force anyone to override deprecated methods.